book_source/01_introduction/03_how_to_use_book.Rmd

```diff
@@ -2,20 +2,15 @@
 
 This book serves as documentation for the PEcAn Project. It contains descriptions of topics necessary to inform a beginner and advanced user as well as requisite materials for developers. It does not contain low-level descriptions of functions within PEcAn. Our aim for this documentation is to educate you about the PEcAn software, the possibilities of its usage, and the standards,expectations, and core workflows for developers.
 
-
-### Main Topics
-
 This book is organized around these main topics:
 
-**Basic User Guide** - Brief explaination of PEcAn, how to obtain the PEcAn VM, and explanation of basic web interface functions
-
-**Intermediate Users Guide** - Explanation of more advanced customizations of PEcAn from the web interface. Includes explanation of PEcAn settings file, advanced web configuration, and handling remote execution of PEcAn.
+**Basic User Guide** - Brief explanation of PEcAn, how to obtain the PEcAn VM, and explanation of basic web interface functions.
 
-**Advanced User Guide - Cmd Line** - Full explanation of PEcAn, tools that are not integrated in the main workflow, and 
+**Intermediate Users Guide** - Explanation of more advanced PEcAn functionality from the web interface. Includes explanation of PEcAn settings file, advanced web configuration, and handling remote execution of PEcAn. (TODO: Also, command line tools?)
```

Comment:
Why are you deleting the Advanced Section?

Reply:
Whoops, this was a mistake. For some reason, I thought we went straight from "intermediate" to "developer", but you're right -- we also have the "advanced users" section. I added it back in.